src/control/server/init/setup_spdk.sh

```diff
@@ -17,9 +17,14 @@ else
 	PCI_WHITELIST="$_PCI_WHITELIST" NRHUGE="$_NRHUGE" \
 	TARGET_USER="$_TARGET_USER" "$scriptpath"
 
-	chmod 777 /dev/hugepages
-	chmod 666 /dev/uio*
-	chmod 666 /sys/class/uio/uio*/device/config
-	chmod 666 /sys/class/uio/uio*/device/resource*
+	if [[ "$_TARGET_USER" != "root" ]]; then
+		set -e
+
+		chown -R $_TARGET_USER /dev/hugepages /dev/uio* \
```

Comment:
(lint) Double quote to prevent globbing and word splitting. [SC2086]

Reply:
Should heed this shellcheck error:

```suggestion
		chown -R "$_TARGET_USER" /dev/hugepages /dev/uio* \
```

And ditto for the `_` before `TARGET_USER`.